MG-264: Update R notebooks to ensure model values align across files

JessterB opened the pull request:
These updates ensure that joins on our `model` display values work as intended:
1. Fix case mismatches & adjust GX expectations
2. Disambiguate the two 5xFAD models

This PR addresses these issues in the `disease_correlation`, `pathology` and `biomarker` source data. The required changes to other source files (`model_info`, `model_allele_info`, `model_results_info`) were made manually.

Note that these new/modified notebooks have already been executed and updated source files are already in synapse.

JessterB committed:
Update R notebooks to ensure model values align across files

JessterB committed:
Fix misnamed variable in pathology notebook

JessterB committed:
Update GX to expect updated 5xFAD display names with center qualifiers

JessterB committed:
Revert unintended change to genotype strings

jaclynbeck-sage reviewed (approved):
Looks good! Just to double-check, there is no Jax 5XFAD data being handled by this code? Only UCI?

JessterB committed:
Add R notebook for disease_correlation source file

JessterB committed:
Add and clean up comments

JessterB commented:
> Looks good! Just to double-check, there is no Jax 5XFAD data being handled by this code? Only UCI?

Jax doesn't generate pathology/biomarker data, so all the 5xFAD data those notebooks handle is from UCI.

UCI doesn't generate neuropath data, so all the 5xFAD data in the `disease_correlation` notebook (that I just added...) is from IU/Jax/Pitt.

I also added a comment to each notebooks to clarify.

jaclynbeck-sage reviewed (approved):
New file looks good! Thanks for the clarifying comments.

beatrizsaldana reviewed (approved):
Looks good to me!
